database/pg_hba.conf

```diff
@@ -0,0 +1,14 @@
+# PostgreSQL Client Authentication Configuration File
```

Comment:
These database/* files made it much easier for me to manage the database with docker, most of the work is done for us. 

See: https://www.perrygeo.com/dont-install-postgresql-using-containers-for-local-development.html

Reply:
you should just be able to do a really basic docker setup with docker compose for local db

Reply:
This is a really basic docker setup,  most of it is default, just includes everything that can be helpful to change down the road and keeps the data local